prisma/migrations/20230827190647_pupil_screening_screeners/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "pupil_screening" ADD COLUMN     "screenerIds" INTEGER[] DEFAULT ARRAY[]::INTEGER[];
```

Comment:
The migrations are not executed automatically yet, right?

Reply:
right, I'll do it manually with this one, and enable it if that worked well